sodym/named_dim_array_helper.py

```diff
@@ -13,3 +15,10 @@ def named_dim_array_stack(named_dim_arrays: list[NamedDimArray], dimension: Dime
     for item, nda in zip(dimension.items, named_dim_arrays):
         extended[{dimension.letter: item}] = nda
     return extended
+
+
+def sum_named_dim_arrays(named_dim_arrays: list[NamedDimArray]) -> NamedDimArray:
```

Comment:
I don't think we need this - since we have the `__add__` method, `sum(nda_list)` should work fine?

Reply:
yes, nice!